src/components/Editable.tsx

```diff
@@ -346,6 +347,19 @@ const Editable = ({
         ),
       )
 
+      if (inputMethod === 'dictation' && newValue.toLowerCase().indexOf(' new thought') > 0) {
```

Comment:
This would result in a false positive when "new thought" is typed on the keyboard and then the user enters dictation mode. The intended behavior is for "new thought" and other commands to only be transformed when the user is dictating.